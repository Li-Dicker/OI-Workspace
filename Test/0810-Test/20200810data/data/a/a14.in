10
200
61221 35284 63296 49949 34760 20144 59037 75410 30103 350 10934 44178 25495 90421 31790 26807 54853 97866 3872 96382 71217 19834 22021 25327 83670 6520 83235 24562 97961 57559 26155 91365 94354 93617 94401 58413 81496 72816 17017 1913 80810 84715 29628 12316 7795 91772 13478 79528 48728 66017 54117 58364 17371 82317 13964 8194 91940 24645 37768 94308 24564 74088 45117 99610 89808 87157 54256 77058 90543 34860 93602 66188 18030 53075 85108 40251 13917 17513 34551 18405 16612 71237 2848 47058 48321 83430 83595 13496 59498 4409 90981 9419 98068 46090 94227 18311 27595 40100 61605 85443 96259 16372 61895 82367 9483 84568 52734 52364 2070 22576 28095 45282 46714 16511 35801 40082 6604 33044 82790 63548 93343 38201 83558 73067 10903 10648 44154 59189 31826 65327 80985 72764 51056 951 49186 58327 34217 47009 81392 63398 45157 17774 38528 17913 4107 52306 61838 39904 94034 95231 78138 449 18548 98656 17131 6495 85359 85093 16426 89875 84195 21479 94994 47984 30496 60865 854 9885 91038 86766 43063 4473 71980 92488 30003 41518 45913 19448 71753 29203 53164 17684 55965 87738 69824 10929 96227 11191 84460 98083 16163 53189 5434 57584 2618 34749 86993 19220 37448 69265 
42162 10121 30497 30923 41056 36210 30376 32875 18388 9062 49168 32948 47067 16219 48679 7193 40300 5876 26265 8375 16924 4532 7597 23270 42672 32505 14685 41627 46572 222 14490 23847 6074 24441 7618 1711 32269 17961 43536 43869 5143 9323 17845 33939 154 33228 2317 30420 26538 35996 27375 34768 24263 3663 42471 25184 6494 19887 29200 49420 13746 6012 48314 7564 41184 35912 30055 34545 6272 42279 30032 22891 14304 24568 23470 16932 1909 32846 402 25261 36048 44747 8111 2118 5102 33402 11180 18159 15829 48206 13955 17086 47511 22112 4717 39702 1304 7158 8161 2728 332 16260 38481 42205 4165 32898 8825 34217 36143 17019 22281 40175 31274 42664 20223 20593 34986 8056 47227 12737 12435 46259 17246 29290 2722 24910 41978 5299 13694 16645 36780 48192 28272 20443 25127 17863 29601 24858 40137 16668 2175 6599 44163 15716 44986 7207 46833 3167 6662 44746 14062 10134 36349 47686 10811 25206 35635 37372 13584 31771 43219 16276 2847 30075 7003 29157 19655 31431 22323 26853 18120 15157 19704 23883 29861 29995 27211 13029 39690 47262 17446 40544 26535 543 27911 12356 25712 34685 49686 15538 2978 44857 26435 3166 40158 18590 47235 26057 42425 22427 
200
68343 87318 81463 65254 68679 43068 79850 95240 24927 98198 96526 84521 27391 78635 24984 57925 96965 60238 94641 67304 56416 8685 5862 47713 54522 12447 28886 79381 4340 22464 43681 805 55014 37868 15988 54582 76119 12667 10891 10317 67035 85779 4767 48097 9948 95921 15514 24047 63126 57670 83137 93062 48103 7220 18286 54871 94022 63933 8658 14323 93546 79244 56461 71825 22593 75364 89574 88424 37196 36782 69403 18896 4825 14287 79191 26244 7763 74326 76151 12848 10301 69968 87992 70865 54434 90057 71569 77941 16663 20779 50348 71737 46368 94528 10681 21069 82129 61424 28146 49795 48619 42935 94695 40488 51766 16674 15352 73309 16579 45472 58719 75664 62793 63568 13937 95286 80312 23998 92244 79078 9645 87974 95540 51973 68429 40903 4192 85402 92148 68986 79083 75912 20119 43955 52568 42084 36016 2758 64730 30605 89737 22925 95295 60915 52858 55751 78706 4997 81493 62840 52630 5759 84975 52443 64381 33947 85120 41795 73903 78557 65377 90773 33861 85934 82868 65686 26915 13789 45011 71027 24578 50779 91909 71073 6634 87153 65276 50797 21476 29036 93965 43981 3593 15084 24791 86421 36295 44038 67612 35341 83197 32338 27577 35868 74308 10338 49962 84809 42332 22594 
19925 17533 5968 14654 4522 22815 4529 20033 32579 25121 20448 6129 3473 26845 21609 36383 46086 19214 21542 11925 28290 29191 42031 24425 49625 44046 39895 11938 43560 44784 10656 1121 44272 35960 19097 3339 33752 21386 37040 16927 15469 47865 856 36406 4568 30515 37485 36799 42602 19243 30891 24569 9099 12016 12617 3649 41817 17883 21293 46062 34836 34723 47047 24172 19135 38128 18517 11948 39398 11602 4211 46182 21435 39207 40902 43978 24807 44663 12213 30548 42347 23416 49774 35291 23541 4489 17633 31661 15685 49333 13508 44944 47639 15515 35526 44245 998 365 6586 5777 35530 46339 34918 5821 17317 280 16047 36179 31014 18682 8492 1179 35267 35623 23640 5600 17728 25992 16348 27529 31164 10575 47514 33849 32259 13838 5355 47283 14968 36892 43567 47315 4841 31559 36011 43897 21680 3963 42427 26553 35225 17529 40744 49499 22332 4963 49336 31446 36102 39422 27309 44819 6097 348 7372 30136 17241 23574 30954 39602 17529 27153 38049 4974 19335 37369 14314 11550 20294 21740 22038 36273 13564 6131 15433 34218 40373 24280 1959 35338 45127 32813 41135 40181 31487 8068 3710 29418 23225 4165 27415 16498 33787 42970 14409 42663 8588 2501 31496 28879 
198
47099 41260 24898 77344 24052 9225 39226 72211 32789 86226 8140 17126 58627 10426 27279 28088 88240 47237 12861 16043 71872 65158 80841 41450 27912 74089 36460 2780 27519 16087 71732 40449 10778 85868 32102 51150 58527 60844 79231 13431 99851 86882 33074 77470 10667 25032 13460 50889 79757 21098 33260 11886 50015 73090 64056 37730 71694 24187 77022 79799 19087 36693 25225 72569 64496 89811 51486 23574 56118 29010 46116 60009 64056 4787 99088 20111 57953 2500 62617 16419 9195 87644 14894 40390 23670 69891 30851 18637 69051 59942 21326 65331 81309 51886 41406 91155 22812 83162 70562 15958 71019 77176 42684 54830 88148 64462 19230 88566 75169 93989 61894 30961 92557 24652 91299 96628 89692 87685 61155 40333 23705 9011 23400 79983 33209 50409 50080 72856 78836 79606 99978 64340 9961 80109 88145 3456 52957 31166 22938 73792 87838 91375 85424 81024 3435 88467 45171 60978 97456 48960 78978 42584 57436 82677 88095 21245 41278 1252 33575 46827 74835 47043 63376 24510 27140 32332 80242 16025 59070 94972 19897 17623 5094 78586 53136 62770 59529 87952 11728 59586 42120 84143 28100 12637 79025 77510 2204 42493 24524 65450 88916 20804 40080 33621 69722 34188 93143 74165 
35397 6449 45339 35925 33493 43958 30166 23036 22818 28761 10406 44630 27577 37939 27211 43417 47245 7908 29266 32151 34884 49219 45249 41923 45864 22539 19087 11848 13703 43477 2044 31172 1517 6424 33055 11100 26909 39798 4407 4410 36109 6023 30640 17268 32680 4077 18275 35481 7172 30126 16239 38665 24456 23875 21376 15435 49584 36795 22998 44564 43376 27747 26381 44572 31427 34700 29686 24186 11720 49657 16426 4120 39527 46926 22749 22228 17400 1472 13358 7024 19778 38234 3934 30844 21197 8770 42951 11968 18858 18144 45784 15639 23630 44661 8296 14863 25982 10930 22997 26262 26739 24939 10878 23596 46009 28389 10045 22621 2983 48097 34731 42185 24195 5697 16698 2530 19431 35658 28792 32557 20438 19371 45919 29580 28107 20820 19340 12766 17131 34118 38358 29376 6270 44979 47977 8335 7468 18010 31352 1927 15463 37063 32921 32731 17185 44964 15063 39940 25929 26962 8741 40351 32099 24301 5317 47101 7005 4848 46622 16908 5729 6683 11836 39093 48479 11179 5143 30117 43076 37895 46143 43608 5737 41158 20897 20405 44117 35684 33005 8960 31900 14437 21964 48804 20590 49835 32457 39938 13617 20230 43087 10427 260 47978 3269 26813 9386 20106 
196
29318 76669 65426 92793 25547 81135 62266 30838 10812 91386 68077 6289 89716 19472 55597 37979 29058 83071 41087 45377 27178 17136 21273 3057 63028 5592 36248 50543 19990 50851 54143 20506 13716 82597 97656 10561 79564 56440 74723 67606 46497 44886 13269 576 65454 9748 34505 97300 29253 1268 97564 83304 9203 42102 10408 17444 35229 67435 54918 53850 93652 77627 30340 49127 44962 55414 27779 99713 21327 35195 81841 50535 55940 88934 18853 16206 23168 64081 80549 32352 12228 25642 80492 2594 78947 92157 46077 79962 17472 15380 730 39691 18539 41307 25896 42805 698 50085 51457 18772 27169 94182 53620 56578 15824 51504 89647 84717 3761 35643 2274 89738 60447 60559 33234 7798 24298 85048 32488 10223 29103 56194 17769 85626 16030 3244 88643 92465 33760 46404 87482 9876 20867 97155 7523 80773 20907 49240 79625 19573 76332 21675 88477 8086 72762 21716 77721 11419 11695 89345 67591 82044 73073 44120 71586 75472 58358 68892 13871 72658 16729 1015 67658 38290 78754 19200 10174 3434 49551 77367 22994 52296 17913 4739 42642 66765 93809 11780 24545 55818 89504 27855 58971 11654 34245 82563 2884 86790 30753 27311 85556 75992 76412 21421 9080 42770 
3939 27226 29795 26673 10071 42956 1863 34233 12040 13808 37999 27777 34917 43237 34737 9994 36489 23377 49594 47342 3156 22123 19936 38498 6784 38529 4195 19147 40118 26466 4507 44867 15202 25514 12121 20679 29486 1218 17931 48531 2284 17663 5803 23018 29064 7745 23153 33293 15862 17123 26660 40226 9107 14139 3830 32778 31015 22098 18848 4122 39522 33560 1252 49076 35291 308 19023 46393 34759 38287 14279 6563 25862 22740 35881 36157 29967 24921 48681 17710 16182 41309 48174 19956 19935 19957 24450 21086 9679 38761 40488 11789 28496 18670 42931 8167 3619 17048 8533 21334 44636 8253 17550 35140 2893 45536 24902 4701 2894 49918 21788 44229 44791 49654 30795 34403 9896 13296 9132 19927 7689 22941 13367 5519 35751 13090 22055 16567 31344 40869 38710 27166 10930 5889 32365 31111 48990 38406 11911 3442 21066 46072 31198 1734 33892 22104 12929 388 14268 24089 32127 20946 47589 29392 47576 35964 30390 44693 35979 43509 45460 46509 14466 11060 36852 4219 15014 32332 39087 44382 41883 45055 23215 3003 6310 38445 45643 15496 8118 44066 44891 16758 25588 25112 19125 30122 7716 25794 41123 39957 682 5137 889 41500 35928 34331 
200
4741 40473 95776 71104 86588 59183 81797 22655 32970 99937 71503 25350 89025 4047 55388 40460 98754 51917 33769 60891 47130 75748 93031 82026 14315 2792 16784 81275 91492 71180 6435 9682 86144 72932 59324 19145 70439 29292 66429 36190 65316 92320 70965 21766 26426 69613 33082 90995 7036 71910 68958 24064 90007 65297 61493 41183 17632 43862 57989 33425 64642 78288 73112 78709 17416 65628 22478 84166 84557 25583 97668 67405 84321 20669 6321 81124 61702 58992 31005 76461 52942 27648 10256 12223 81436 98645 57716 17280 56447 73883 99313 4595 83952 69839 71168 90490 63451 22819 13675 66509 89366 2082 23846 69775 11511 13110 23134 4969 10214 48381 21339 31298 12920 69414 49728 73416 58952 36316 3762 4772 65523 1435 92817 63309 40212 94064 8691 20429 77806 75275 34855 30853 60002 30874 76905 9787 86473 20884 85023 32193 41562 92511 23730 44166 48724 67240 9308 6546 74801 27505 13740 43862 54815 13682 72490 66785 7839 17266 20264 68745 95984 63397 78324 44777 63647 95373 44680 37539 65179 39334 30794 60979 514 33113 77595 56246 69588 71423 81063 25196 68214 27676 6335 92858 91872 15843 92076 80184 41481 58290 81208 14219 66483 81808 82166 2349 40972 2127 91882 19518 
45766 48029 1332 12625 4218 23794 3032 26365 4271 49389 29591 25225 11184 21979 23 18978 7999 14958 47729 32429 36306 12846 32938 16137 18388 31964 19368 34138 659 38459 9702 5238 18905 44900 32566 15236 45582 994 4680 47577 7075 47061 33592 29648 47892 26173 28437 1442 46912 5236 30011 23491 26046 525 44553 23188 29934 9131 46229 9553 41197 33455 35134 43724 16263 7162 21765 27928 19965 19288 18820 8249 42803 38863 18487 12655 37178 1085 49005 21687 33321 24948 3025 4064 48275 9227 1057 25992 9083 10714 20201 6059 40368 18757 34102 14240 8895 32914 18594 9175 49107 17850 6516 18933 2525 11561 8443 38572 28397 46556 16224 2343 29679 40173 21951 35020 31661 3989 46985 14078 4116 17939 33380 16956 45434 49716 27336 1444 31999 34117 27232 31594 4769 31343 13278 35668 6938 11196 47949 9326 13276 42096 23567 42881 35996 38682 29245 667 12779 17054 42703 20715 13009 23058 38883 21404 14354 23773 45169 19340 44213 13684 3527 17775 25923 45827 36056 39352 47113 4180 1869 18893 25634 35847 37950 15931 7122 30320 34570 13295 48530 27576 10974 38548 18179 22831 28808 29938 16110 11814 49028 47767 35550 30460 25124 44820 20164 11499 3123 48443 
199
39689 9318 6242 12301 9199 78874 20828 13563 79559 49091 34757 98030 96266 42018 79037 1296 46317 25955 38517 79495 99475 16469 37674 12337 5760 63912 1081 23790 43271 90353 68342 53787 32263 76846 76241 57714 39853 23732 12969 51854 89766 9974 6781 52132 99547 17053 85034 68968 95146 70471 45007 78560 94149 52576 50540 96530 94111 89123 76575 91848 70692 24975 53526 4822 27951 20575 77111 95234 80339 89549 33202 68530 76681 15168 51995 75031 23204 53288 77194 42655 15654 99865 14890 43066 8369 60904 50454 1537 13393 31527 48494 10841 79411 43024 69488 11413 4130 53520 12442 79004 21398 88006 38003 52552 49488 49001 53738 28362 24251 37098 52223 87709 93417 28589 63314 38730 42755 31925 69323 95131 46539 14735 32705 76615 36319 44129 86680 38685 46131 3387 72809 48475 68486 17773 67458 16029 68210 43277 31012 19513 17298 38485 19855 79132 87494 92549 54582 13199 10954 70827 58788 90857 28981 82125 33758 42592 29757 65282 62541 7646 95185 48727 32377 88893 79574 23304 81422 44562 9261 80643 34389 10119 29951 38764 7882 11679 11165 85227 36608 25451 77441 849 83525 18656 9187 22842 3663 73532 28296 38881 60767 55715 2155 86159 11125 75854 63495 70499 56110 
21938 10615 8470 19677 33302 21246 31810 8123 13483 29137 3402 15573 2388 49347 24664 5474 21275 21237 19445 22103 46747 4162 10536 17251 43119 15353 20925 47398 35962 49386 48440 18382 7588 31869 23613 45960 21344 12749 22930 26061 2202 26874 42923 43208 11803 33513 3069 14934 11427 36964 19825 24370 45278 23937 40677 38771 12137 2090 496 40060 24512 575 45911 3308 12280 49922 49716 31095 38491 19491 25127 288 37413 9061 20927 28121 5537 33856 37820 19630 1374 49644 7278 49241 35730 38746 7277 42767 22526 3064 30509 47224 33624 19440 45137 46896 20626 49184 19990 16390 712 702 48158 15585 48882 25282 14083 34400 36783 34080 31795 36747 23640 5380 1876 9070 44400 20933 36325 46065 21983 13742 2441 24651 12370 9032 48279 21839 22361 12342 44208 3849 9840 44958 9353 36356 32215 43747 34919 9200 10127 13100 42667 48983 24889 43795 14087 34840 1769 19403 25829 46533 39578 37496 29518 24729 11551 21011 7519 19465 38312 26597 22814 29965 31846 23907 16771 44264 15482 29574 28702 41770 42599 10854 31246 40753 34527 29644 35752 41972 34137 3381 26611 1694 5774 44956 40802 31173 25427 27360 10902 35887 9581 46682 12462 9486 38911 48172 8094 
196
34723 58900 77949 20916 12301 64200 67912 26870 99098 42252 88508 84958 6850 66395 74274 57550 91613 27062 94214 74159 66289 39479 54368 27895 8395 99519 54569 36956 21982 65114 97497 40915 35986 59829 28352 73779 45705 93546 18606 37321 5084 3631 31443 10095 76408 49373 72518 66791 15596 76487 11524 60973 57062 70122 5019 23216 81938 60352 53880 76392 83365 36358 6594 15974 59798 72651 19909 69662 99357 90960 75310 44660 62020 65269 13430 67799 68795 57430 22599 34262 26572 99288 89837 446 85969 60867 82434 35863 46652 16302 89554 50203 30156 56395 5269 99683 17692 80331 43862 50014 93355 98811 68918 18669 3609 1213 80668 85558 67487 89554 87704 86350 73676 90004 28576 42224 48517 67322 99659 79941 77224 15151 14287 64670 85113 29030 92594 24940 39308 49456 42313 34850 5392 48134 24049 69839 12732 67440 31860 74452 94999 26659 37256 54033 99786 92316 78039 81749 28918 61486 51683 75816 31944 17691 36679 2941 13393 12288 85712 34301 6774 97440 13246 34546 55567 79603 43876 4078 24790 1850 14885 61973 31423 67736 98293 29247 22949 62919 77172 35896 26059 23314 51554 67003 19947 38934 43874 686 39436 16716 16480 58968 82497 92551 28150 34653 
10860 39281 26895 29816 5340 26405 7226 39028 46637 41844 24410 14195 15350 24994 21066 2466 10161 40275 33669 34975 33725 18552 1751 15837 5474 7860 33592 41917 18556 48614 6538 39726 11217 10583 2551 31307 25909 22529 15232 6063 1990 22292 49034 28316 10653 49265 15990 1899 15272 38180 882 41167 15285 40186 10915 391 49314 37449 21286 46943 5547 29322 8043 2612 26989 30692 44910 20648 21439 7193 33446 1251 7806 42857 28900 27189 20616 35981 18179 8263 24849 21959 15046 5204 26721 47802 21612 4496 22323 10437 47968 22936 28354 14365 46375 12808 35941 1356 35852 3032 13840 28013 11412 46003 42885 23883 12511 42208 29560 13770 1954 8164 37704 47174 2559 36098 18921 11759 40594 14373 24350 30632 3813 22334 12641 39303 30737 18619 17233 2583 47986 36746 6395 37443 33114 32050 36048 15059 7972 4787 22869 33004 8567 2303 4043 19692 25675 8692 28631 28978 14958 33502 22497 40819 42811 49298 40006 6898 48872 16954 26585 22005 497 3565 30366 44098 2858 48036 36447 2266 7550 25970 9700 45777 41989 43103 49216 12309 13908 13344 20856 2582 27992 32570 30726 12038 6554 48333 20301 34061 16166 3001 43158 9977 14248 33825 
198
89115 56859 82486 63651 48177 93180 9000 44955 47429 28209 10939 14920 75142 73174 55293 72344 42929 13736 67655 36103 84909 91797 50374 86578 83699 11635 98719 33111 50083 38324 51566 76186 60547 64124 99861 15820 3951 87370 63067 38830 87730 28277 41989 16846 12999 9257 77393 5156 42698 11688 42072 31322 81500 97237 6002 21476 58896 46718 36591 26538 53439 17373 1432 2972 77285 53798 86866 42161 13699 65175 36605 60982 75306 61993 9633 30978 39668 15464 66499 18355 4911 50387 26969 91903 47706 87442 43571 98915 97179 56065 24552 41785 27137 86138 79446 46213 41558 13870 29284 96366 48933 29632 88057 47398 40818 13006 77724 84020 24003 98930 50687 86475 66010 55390 88603 12499 28757 39195 95255 30310 6318 36582 32953 48217 18838 91033 90790 19723 97989 74716 39409 9011 21872 65704 95602 50474 6972 91462 24810 12326 56296 19323 13914 33224 83882 35880 76850 66334 15678 80106 47427 81281 81071 35998 48539 23152 25331 67143 62084 58770 79895 89861 39154 2702 71248 78123 19363 75096 9489 84372 11367 39241 39864 11488 51491 67604 65563 48519 23517 48079 79726 86956 70091 28794 98119 13375 76126 32577 25348 93651 60671 80551 35288 22464 35752 11395 43742 53533 
26676 15183 12231 9619 45947 3439 22720 26822 31670 13203 30738 29793 8043 29803 16793 46016 22199 45944 17846 48173 40895 43697 8191 34091 9043 10813 39020 38279 19255 1412 49985 37398 24408 28156 23570 44175 43437 12535 472 43551 18750 34354 38063 10102 15117 34412 29305 39307 862 19121 37354 18776 40180 23564 32408 22644 14022 30974 37720 47607 47252 7067 3641 34868 21535 30668 46120 44516 44976 21452 43765 38831 20521 27461 8164 40404 18274 17986 34989 2407 25712 2847 28126 32896 40993 49116 7767 42304 8393 8200 48209 21891 18769 48535 18798 2989 11276 39543 16858 14041 10695 15322 11024 37762 39143 26219 37009 9618 28984 12509 18443 31827 18894 39810 10604 3926 5861 14390 26439 36101 32595 28305 24754 38335 47393 19212 38684 45152 4059 13698 13157 1310 5517 17162 45117 37188 21983 21954 26443 36294 806 12076 15989 3697 883 20969 2136 14220 12821 33473 36802 31555 27091 30192 34344 3763 1496 40757 26849 41632 21055 32377 32714 8301 16108 32453 31280 37375 13598 44966 39667 40699 2815 31437 46692 6925 17295 34199 23166 28870 29214 9754 29326 49649 35436 46887 32943 26724 33947 3777 15801 20982 43040 35845 26646 47739 7388 10745 
197
22118 30156 84853 87314 53218 96419 84409 78230 90741 69142 74887 92141 67725 91318 70858 27656 88591 70990 87653 27113 78732 8176 35245 3603 82320 42209 52721 59020 80576 96008 43125 52071 27159 34977 14458 96848 38487 69118 91590 32240 85249 3261 96583 99298 82519 50726 9973 18872 13794 74169 14714 75482 11681 82127 5169 73691 50641 27693 70341 57941 35850 13269 19466 51092 36912 56232 66715 77349 14323 48097 53184 3975 5927 46020 99487 54796 85939 84741 17450 82767 62958 40069 74250 21065 12707 47374 17343 11803 87639 47260 72845 77843 87256 64253 14771 50535 70839 72504 12844 69036 23449 58329 6543 47278 20437 34331 42527 22707 10109 99300 51129 93638 25912 22700 17968 84571 70552 35407 20691 74905 5315 40945 94591 23761 40649 32006 62310 98466 68172 13246 17796 35260 64484 1671 33133 63453 57411 50876 6084 50728 8588 21792 14632 9427 20680 92465 8881 63573 23117 95785 41007 46468 73252 4700 55223 82674 57784 21028 6666 90393 15603 24274 51095 83474 17990 6474 6451 70198 24018 27509 52448 65102 46267 64914 31351 68306 946 32778 67611 44314 26379 20472 53242 83474 69003 73502 30303 90702 78981 12686 82791 77758 11162 66113 7094 37521 44762 
16654 21638 48348 438 36948 17031 29113 17879 46463 3189 7228 14166 31604 34874 47438 15231 22718 25855 8671 42969 34913 20032 12391 46665 2547 12865 10497 10616 29245 24841 29257 26958 27327 6373 8208 2560 18809 20217 21699 15320 645 26398 32604 20387 30657 17343 12805 27450 39267 17344 33881 24946 47143 32838 11972 30363 41798 24276 44073 38013 14860 7106 39763 48588 47301 28062 8796 17906 47400 41451 16447 46247 24258 48786 22017 26640 21921 33701 30392 34627 29520 964 37490 29276 20860 11986 43833 46703 37307 46631 37668 43902 26751 34655 12913 2592 14346 46395 20308 33052 43967 17634 43559 7215 42320 7659 48445 31580 5956 21935 36249 20562 39878 29170 27975 33932 24889 29542 29523 18103 10302 4093 27653 12104 17872 47868 29273 18659 24193 25849 16030 37664 34545 15346 15978 2412 39173 26462 28604 5888 34997 38937 25751 15639 21581 45418 16902 49627 48711 7595 31316 38725 9943 45971 40795 17114 47389 22985 49030 37495 19428 41649 20973 35550 40380 19793 26327 33208 36721 11430 44774 13757 18901 49227 19413 45731 35148 6976 33096 15331 6335 32050 9759 38591 26060 24378 32114 45866 32370 16203 46513 32553 42665 11390 47427 5379 49925 
200
87937 63989 9011 98537 14981 4968 96606 82576 5927 73195 66769 16081 23708 42967 73276 94801 68641 33426 18037 96061 95042 75213 64209 54966 60365 61048 58486 66631 53392 10430 21337 13444 51597 99012 42996 99829 52426 65969 95424 92776 81759 91065 17918 33712 92178 104 7874 8172 15775 20780 95315 53341 75680 61892 90503 87979 2435 84100 50782 59213 96662 11683 17812 58029 19152 16548 40565 36413 25582 7102 84591 98269 930 79262 23710 90281 33487 23591 98674 2735 59139 13936 10279 94594 5740 17712 99748 83205 95245 97756 60859 41461 10460 1426 83212 82180 38163 29430 27996 42513 33203 5422 17266 5906 82337 49654 85130 61935 45431 27344 66781 81017 46050 64990 7541 62220 26639 95136 79326 24620 64183 94356 1277 35528 36923 37904 5636 8416 3653 34093 20267 72090 19840 92182 54559 1119 21720 6154 85093 42877 52313 81181 13554 9879 50325 7980 36268 87306 23889 30251 69138 71370 82105 12035 95916 35394 3489 95766 97998 34450 59807 72772 15948 28277 54858 82259 79032 92456 72422 90022 59050 68052 50244 33465 92868 29678 24342 29467 6621 46702 71659 46116 63304 71285 5486 6988 90103 27081 88441 66021 56834 44976 5026 38526 90494 97822 46646 69080 94683 3154 
30028 17560 41284 14204 22304 30303 8224 4182 47415 46302 11742 3269 35397 14092 30104 29357 28393 10659 28863 4670 30536 8829 23378 18507 40252 2392 43634 42540 12166 45921 15017 23857 29433 21561 43807 41708 14591 6129 20787 40801 27756 44792 7997 24301 20567 9882 48347 5056 11332 4597 19518 37257 29286 49708 30451 38606 6187 43851 47385 9975 33188 11942 42729 7239 8500 7190 15953 2226 41846 7087 46226 47084 41871 28544 40523 27238 11891 4269 31116 48123 25637 31934 20986 31005 5372 24385 16416 25856 18304 39253 47179 48192 10320 34907 965 20970 37508 35229 49042 23460 7095 28104 28628 5227 11096 32862 1970 28421 27574 1168 2118 35575 47098 11748 22164 37777 4587 6436 48106 11731 2031 30782 22250 14875 32814 18009 35852 695 873 3735 23364 22085 41298 6196 13806 34984 36722 31976 40924 31748 45143 16887 14408 39922 45294 40009 40204 28344 27567 7323 36143 7381 10774 46814 47045 10135 5426 31370 26301 24438 34411 6256 14700 19898 28549 11125 30313 45082 40084 12888 10755 25608 24869 44292 2410 929 23968 3025 32653 42152 40455 20969 478 2210 16844 29532 10096 19940 8106 40600 49835 41961 16303 8083 12645 49109 28089 31608 34958 16323 
