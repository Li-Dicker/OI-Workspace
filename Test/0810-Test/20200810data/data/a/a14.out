Case #1: 13882672
Case #2: 14919133
Case #3: 14453348
Case #4: 13175943
Case #5: 14044652
Case #6: 13884875
Case #7: 13643250
Case #8: 14339600
Case #9: 14425560
Case #10: 14212901
